### DISCUSSION TITLE
Support Redis Enterprise Active/Active with BullMQ

### DISCUSSION THREAD

**Discussion by jphaugla** [taskforcesh/bullmq#1727] [Show and tell]
I am doing some testing with bullMQ on Redis Enterprise Active/Active and not seeing any problems.  I work for Redis and am partnering with licensed BullMQ users wanting to run bullMQ with Redis Enterprise Active/Active.  They mentioned I should contact Manuel Astudillo.

**Comment by manast** [comment]
@jphaugla If you want to contact me you can do it at manast@taskforce.sh
